### PR TITLE
Feat: Add home page

### PR DESCRIPTION
**Pull request by starver20**
Added home page for quiz app
Added 2 categories of quiz.
[CSB](https://sbbm3.csb.app/) for reference

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+@media (max-width: 770px) {
+  :root {
+    font-size: 40%;
+  }
+}
+@media (max-width: 660px) {
+  :root {
+    font-size: 30%;
+  }
+} */
```
Comment (VishalPatil18) [inline style.css:71]: You could avoid commented code in the PR.

**Review by VishalPatil18 — APPROVED** [review]
Just one suggestion was added. The rest looks neat and clean. Great work @starver20 👍

File: style.css (hunk 1 above)
Comment (starver20) [reply style.css:71]: Thank you. Actually i want to use it later, thats why i did not remove it for now.

**Review by mohammed-sas — APPROVED** [review]
pr good to merge. clean ui

**Pull request merged**